lib/migrate.js

```diff
@@ -0,0 +1,444 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-couchdb2
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+'use strict';
+
+var async = require('async');
+var assert = require('assert');
+var g = require('strong-globalize')();
+var util = require('util');
+var _ = require('lodash');
+
+module.exports = mixinMigrate;
+
+function mixinMigrate(CouchDB) {
+  var debug = require('debug')('loopback:connector:couchdb:migrate');
+
+  /**
+ * Perform automigrate for the given models.
+ * - destroys model data if the model exists
```

Comment:
typo: destroys ---> destroy

I am not sure which one is better, but the three lines should be consistent